library/channel.c

```diff
@@ -134,14 +138,14 @@ int ziti_channel_connect(ziti_context ztx, const char *url, ch_connect_cb cb, vo
             ZITI_LOG(ERROR, "should not be here: %s", ziti_errorstr(ZITI_WTF));
             return ZITI_WTF;
```

Comment:
ch_name was malloc'ed but not free'ed here. probably no big deal considering that it's in this else...